bin/main.rb

```diff
@@ -0,0 +1,26 @@
+require_relative '../lib/game'
+require_relative '../lib/player'
+require_relative '../lib/board'
+
+puts "Welcome to Tic Tac Toe Game"
+
+puts "This game is played by 2 players, player1 uses 'X' and player2 uses 'O'"
+
+puts "Enter name for player 1"
+player1 = gets.strip.capitalize
```

Comment:
Any reason to use strip over chomps? No objections against it, just curious :smiley: